environment/prod.sh

```diff
@@ -0,0 +1,14 @@
+# To connec to the production MongoDB instance you must
+# 1. Connect the osu VPN
+# 2. Run this script using `. ./environment/production.sh`
+#    Without the leading `.` the variables will be set in a sub shell,
+#    not your currently running shell.
+# 3. Start the server `npm start` `node server.js` or `nodemon server.js`
+
+# Set env variables
+export MONGODB_HOST='classmongo.engr.oregonstate.edu'
+export MONGODB_NAME='cs290_fortunej'
+export MONGODB_USER='cs290_fortunej'
+. ./environment/prod-password.sh
```

Comment:
This is that file you need to create

Reply:
Easy enough, so we have to run this file manually for each session?

Reply:
Unfortunately yes. I can't think of how to set env variables without manually running this. The problem is scripts that run run in different shells without the `.` prefix, and so the env variables get set there, and then that sub-shell gets killed.